cmd/metricswatcher/metrics.go

```diff
@@ -5,29 +5,29 @@ import (
 	"time"
 
 	"github.com/mattermost/mattermost-load-test-ng/cmd/metricswatcher/config"
-	"github.com/mattermost/mattermost-load-test-ng/cmd/metricswatcher/prometheushelper"
+	"github.com/mattermost/mattermost-load-test-ng/coordinator/performance/prometheus"
 
 	"github.com/mattermost/mattermost-server/v5/mlog"
 )
 
-func checkMetrics(errChan chan error, configuration *config.MetricsWatcherConfiguration) {
-	prometheusHelper, err := prometheushelper.NewPrometheusHelper(configuration.PrometheusConfiguration.PrometheusURL)
+func checkMetrics(errChan chan error, config *config.MetricsWatcherConfiguration) {
+	helper, err := prometheus.NewHelper(config.PrometheusConfiguration.PrometheusURL)
 
 	if err != nil {
 		errChan <- err
 		return
 	}
 
 	for {
```

Comment:
Unrelated to this PR. But we are never exiting from this loop when this is run in a goroutine.